Components/Card/card.css

```diff
@@ -1,2 +1,217 @@
 @import url("/Common_CSS/base.css");
 
+/*Card with Badge*/
+.badge-card{
+    width: 300px;
+    margin: 1rem;
+    cursor: pointer;
+    border-radius: 10px;
+    border-bottom: 4px solid lightgray;
+    border-right: 2px solid lightgray;
+    border-left: 1px solid lightgray;
+    transition: transform .2s;
+    position: relative;
+}
+
+.badge-card::before{
+  position:absolute;
+  content:"";
+  background: #3949ab;
+  width:1.6rem;
+  height:1.6rem;
+  transform:rotate(45deg);
+  top:1.8rem;
+  right:-0.7rem;
+  z-index: -1;
+}
+
+.badge-card::after{
+  position:absolute;
+  content:"Trending";
+  color:white;
```

Comment:
you can use color variables  !

Reply:
Thanks for suggestion.
I have implemented it.